coffee/directives/fixtable.coffee

```diff
@@ -76,15 +84,29 @@ angular.module 'fixtable'
 					scope.loading = newValue
 
 			# get new page data
-			getPageData = ->
+			getPageData = (reload = false) ->
 				cb = scope.$parent[scope.options.pagingOptions.callback]
-				cb scope.options.pagingOptions, scope.options.sort, scope.appliedFilters
+				cb scope.options.pagingOptions, scope.options.sort, scope.appliedFilters, reload
 
 			# provide methods to page forward/back in footer template
-			scope.nextPage = ->
-				scope.pagingOptions.currentPage += 1
-			scope.prevPage = ->
-				scope.pagingOptions.currentPage -= 1
+			do setPagingActions = ->
+				if scope.options.pagingOptions?.type is 'prevNext'
+					# does not use page numbers
+					scope.nextPage = ->
+						scope.options.pagingOptions.processingPage = true
```

Comment:
Is this used anywhere, and how does it prevent next/prev spamming? I assume it's being used to disable controls in the template?

Reply:
This is primarily used in the consuming app. The method to go to the next or previous page lives in fixtable, so it sets processingPage to true. The consuming app where the api calls live changes the value back to false at the point of resolving or rejecting said calls.